pyansys_sphinx_theme/static/css/theme.css

```diff
@@ -179,3 +179,84 @@ p.rubric {
     margin-top: 0.75em;
     margin-bottom: 0.75em;
 }
+
+
+/*
+#############
+*/
+table {
+        width:100%;
+        max-width: 100%; 
+        border-spacing: 0;
+        border-collapse: collapse;
+        overflow: hidden;
+        z-index: 1;
+        font-size: 16px;
+        text-align: center;
+        
+        /* Disabling scroll bars */
+        overflow-y: scroll;
+        scrollbar-width: none; /* Firefox */
+        -ms-overflow-style: none;  /* Internet Explorer 10+ */
+        }
+
+
+ /* Disabling scroll bars */
+table::-webkit-scrollbar { /* WebKit */
+    width: 0;
+    height: 0;
+}    
+
+/* Headers and data cells format */
+    td, th {
```

Comment:
I think you'll need to do something like `table.dataframe > td, th` here, to get only td or th elements who have a parent `<table class="dataframe">` element.

Reply:
I'm trying now